testing/src/CMakeLists.txt

```diff
@@ -13,5 +13,9 @@ set(test_sources
 
 gz_add_component(testing SOURCES ${sources} GET_TARGET_NAME testing_target)
 
+# In general, this shouldn't be necessary as the downstream test is pulling in the 
+# gtest includes.  We do this here to keep the ABI checker happy
```

Comment:
```suggestion
# gtest includes. We do this here to keep the ABI checker happy
```

Reply:
I ended up removing the CMakeLists part, as it has no impact on the ABI checker.